ansible/roles/runner/tasks/setup.yml

```diff
@@ -67,3 +67,10 @@
     name: redis
     enabled: true
     state: started
+
+- name: configure size of systemd journal
+  lineinfile:
+    path: /etc/systemd/journald.conf
+    regexp: '^SystemMaxFileSize'
```

Comment:
Since `SystemMaxFileSize` is commented out by default (and in the proper section), I'd use `regexp: '^#?SystemMaxFileSize'` to find the right place in the file.

Reply:
TBH I'm indifferent to it. Personally, I prefer to keep the config files with the original data and "override" the options. However, if you prefer this way I'm ok with it as well.

Reply:
My concern is that this `SystemMaxFileSize` has to be in the `[Jounal]` section of the file. Since this is currently the only section, we can get away with just appending the value to the end of file, but this seems to be a bit more future-proof in case additional sections were added.